src/components/layout/about/contact-person.tsx

```diff
@@ -29,36 +30,17 @@ import {
     linkedIn,
   }) => {
     return (
-      <Flex
-        sx={{
-          flexDirection: 'column',
-          alignSelf: 'center',
-          mb: ['20px', '60px', '60px'],
-        }}
-      >
-        <Grid
-          templateColumns={{ base: 'auto', md: '459px 459px' }}
-          sx={{
-            display: 'flex',
-          }}
-        >
-          <Box sx={{ width: '459px' }}>
-            <AspectRatio ratio={459 / 527}>
+          <Box justifySelf={{base:"none",sm:"center",md:"none"}}>
+            <AspectRatio w={{base:"100%", sm:"459px"}} ratio={459 / 527}>
               <Image
```

Comment:
Looks like you'll need to run some pretty commands :D

Reply:
We don't have prettier in this project as of yet, but I will be sure to pop up a PR for it ASAP